test/e2e/deployment.go

```diff
@@ -317,3 +322,71 @@ func testRolloverDeployment(f *Framework) {
 	newRC, err = deploymentutil.GetNewRC(*deployment, c)
 	Expect(newRC.Spec.Template.Spec.Containers[0].Image).Should(Equal(updatedDeploymentImage))
 }
+
+func testPausedDeployment(f *Framework) {
+	ns := f.Namespace.Name
+	c := f.Client
+	deploymentName := "nginx"
+	podLabels := map[string]string{"name": "nginx"}
+	Logf("Creating paused deployment %s", deploymentName)
+	_, err := c.Deployments(ns).Create(&extensions.Deployment{
```

Comment:
Can you also add a test for creating a deployment that's not paused, and then update it to be paused, and the deployment should be paused after the update.